src/accelerate/commands/config/config.py

```diff
@@ -76,7 +76,7 @@ def config_command_parser(subparsers=None):
 
 def config_command(args):
     config = get_user_input()
-    if args.config_file is not None:
+    if args.__dict__["config_args.config_file"] is not None:
```

Comment:
Note that this looks very ugly right now, it will be fixed by https://github.com/huggingface/accelerate/pull/853/files